samples/Hardware.Esp32.Rmt/InfraredRemoteReceiver/InfraredRemoteReceiverSample/NecSignalDecoder.cs

```diff
@@ -0,0 +1,44 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace InfraredRemote
+{
+    /// <summary>
+    /// Nec protocol signal decoder.
+    /// </summary>
+    public class NecSignalDecoder : Decoder
```

Comment:
if you'd like to move the decoder to the IoT Device repo, this can be moved as well. Other protocols can come later.

Reply:
Yes, and I also think this can be its own nuget package for nf that contains some protocols that the community have implemented. The structure is very useful!.